usersync/syncer_test.go

```diff
@@ -779,3 +783,32 @@ func TestSyncerChooseTemplate(t *testing.T) {
 		assert.Equal(t, test.expectedTemplate, result, test.description)
 	}
 }
+
+func TestSyncerForceResponseFormat(t *testing.T) {
+	testCases := []struct {
+		description       string
+		givenForceType    string
+		expectedForceType string
+	}{
+		{
+			description:       "IFrame",
+			givenForceType:    "b",
+			expectedForceType: "b",
+		},
+		{
+			description:       "Redirect",
+			givenForceType:    "i",
+			expectedForceType: "i",
+		},
+		{
+			description:       "Empty",
+			expectedForceType: "",
+		},
```

Comment:
What happens if the bidder provides an invalid forced response type? Is that a validation error on startup?

Reply:
Good question, right now it just returns an empty string, but this should be handled better. I think adding validation logic makes sense. I'll update this

Reply:
I recommend removing the `ForceResponseFormat` method and to use that value if it's not an empty string. You can add validation code at startup to ensure the value provided by the bidder is valid (a lower case "b" or "i"). There is likely already bidder validation at startup that you could add on to.